lib/api.ts

```diff
@@ -60,9 +61,10 @@ function makeSharedAPI(channel: Channel, data: ConnectMessage): BaseAppSDK {
   const currentLocation = data.location || locations.LOCATION_ENTRY_FIELD
 
   return {
+    cma: createCMAClient(ids, channel),
     cmaAdapter: createAdapter(channel),
     location: {
-      is: (tested) => currentLocation === tested,
+      is: (tested: string) => currentLocation === tested,
```

Comment:
lol i almost fixed this too 😄

Reply:
i also almost added a `get` method here that _just returns the freaking string_!

Reply:
That's what I was thinking too! Don't make me guess what location I'm in haha